docs/orchestrating-elastic-stack-applications/elasticsearch/bundles-plugins.asciidoc

```diff
@@ -106,4 +107,4 @@ spec:
             set -e
```

Comment:
```suggestion
            bin/elasticsearch-plugin remove --batch repository-s3
            set -e
```

Reply:
Looks like the very same to me. Usually, the `set` stanza are at the very top of the script, so my suggestion to explicitly shows the `remove` command as being forcefully marked as successful whatever happens with it.